src/buffer.hh

```diff
@@ -144,6 +144,7 @@ public:
     size_t         timestamp() const;
     timespec       fs_timestamp() const;
     void           set_fs_timestamp(timespec ts);
+    size_t         fs_hash() const;
```

Comment:
Expose fs_status itself here.

Reply:
I implemented this function because the timestamp is also part of `fs_status`, but didn't want to replace `fs_timestamp()` with `fs_status().timestamp`. Should I do that? Or leave `fs_timestamp()` and add `fs_status()` (would look weird)?

Reply:
Just expose `fs_status` and use `fs_status().timestamp`